src/lightly_train/_task_models/dinov2_ltdetr_object_detection/task_model.py

```diff
@@ -252,13 +253,129 @@ def predict(
         x = transforms_functional.resize(x, self.image_size)
         x = x.unsqueeze(0)
 
-        labels, boxes, scores = self(x, orig_target_size=(h, w))
+        labels, boxes, scores = self(x, orig_target_size=torch.tensor([[h, w]]))
         keep = scores > threshold
         labels, boxes, scores = labels[keep], boxes[keep], scores[keep]
         return {
-            "labels": labels.squeeze(0),
-            "bboxes": boxes.squeeze(0),
-            "scores": scores.squeeze(0),
+            "labels": labels,
+            "bboxes": boxes,
+            "scores": scores,
```

Comment:
Why is the `squeeze` not needed anymore? There is still `x = x.unsqueeze(0)` on line 254 and the method can only process a single image at the time